core/graph/src/main/resources/ExportToOtherAuthorisedGraphOperationDeclarations.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This should not be in src/main

It is only example configuration, users will need to create their own version of this file with their own authorisations.